src/util/nettest.js

```diff
@@ -48,3 +48,107 @@ export const getDeckIcon = (iconName, other) => {
     <i className={className} />
   )
 }
+
+export const MLAB_LOC_TO_COUNTRY = {
```

Comment:
Importing all the IATA airport codes I fear is going to lead to the filesize increasing quite a bit, which we should probably avoid if possible.

Having something like:

```
countryToAirport = [
["IT", ["CIA","FCO","MXP"]],
["DE", ["TXL"]]
]
airportToCountry = {}
countryToAirport.forEach((row) => {
   const { countryCode, airportCodes } = row // not actually sure this is valid JS
   airportCodes.forEach((airportCode) => {
      airportToCountry[airportCode] = countryCode;
   })
})
```

Would allow us to preserve fast lookup, but at the same time save up a considerable amount of space in the final file size.